### DISCUSSION TITLE
vscode tester needed move to chroot systemd project

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4757] [Polls]
the test now run after days of saying I'm spamming microsoft, issues out of scope some folks are helpful
crouton did a great work 8 years ago. i working on a Multiuser - work station install Pengiun android studio and Vscode
arm64 still fails x86_64 builds

now working on systemd and crouton /usr/local/bin scripts for update the newest Xserver config
Crouton did hack it , it still works with the X hack
![Screenshot at 2022-05-31 12-57-33](https://user-images.githubusercontent.com/46386261/171277314-007bfaf2-be21-405c-8517-2ddb8db9cf22.png)
[npm-sh.log](https://github.com/dnschneid/crouton/files/8809169/npm-sh.log)

**Comment by TomTravis** [comment]
vscode is up
![Screenshot at 2022-05-31 19-43-01](https://user-images.githubusercontent.com/46386261/171317695-b190935c-2617-479e-8d61-cd25bea1917c.png)

**Comment by davidahoward** [comment]
im wondering whether i could run vscode + nordic nrfsdk
on my now eol acer (as of june, no more updates)

**Comment by TomTravis** [comment]
i do not see why not, try it, it should run vscode does not run via crouton arm64 still work on cross compile

checkout Mrchromebox this may help you run a more capable linux install on your acer, it helped me Lenovo s330
